lib/Doctrine/ORM/EntityManager.php

```diff
@@ -915,4 +912,26 @@ public function hasFilters()
     {
         return null !== $this->filterCollection;
     }
+
+    /**
+     * @param int $lockMode
+     * @param ClassMetadata $class
+     * @throws OptimisticLockException
+     * @throws TransactionRequiredException
+     */
+    private function checkLockRequirements(int $lockMode, ClassMetadata $class): void
```

Comment:
Nit: we use a space between `)` and `:`. Not important though.

Reply:
Hmm, leftover from original PR.